src/containers/MapContext/reducer.js

```diff
@@ -28,6 +29,7 @@ export default (state, action) => {
       return {
         ...state,
         location: action.payload,
+        loading: true,
```

Comment:
Shouldn't `RESET_LOCATION` set `loading` back to 'false'?

Reply:
I have decided to move the loading event to a separate action and remove it here.